pypdf/_doc_common.py

```diff
@@ -1138,16 +1138,16 @@ def remove_page(
         Remove page from pages list.
 
         Args:
-            page: int / PageObject / IndirectObject
-                PageObject : page to be removed. If the page appears many times
-                only the first one will be removed
+            page: int / PageObject / IndirectObject.
+                int: Page number to be removed.
 
-                IndirectObject: Reference to page to be removed
+                PageObject : page to be removed. If the page appears many times
```

Comment:
Why adding a whitespace before the colon here?

Reply:
Did not see this; will remove the space.